Assets/Scripts/Enemies/Enemy.cs

```diff
@@ -147,80 +145,79 @@ public class Enemy : MonoBehaviour {
             return false;
         }
     }
-  
+
     public void ApplyStatusEffect(StatusEffect statusEffect) {
         activeStatusEffects.Add(statusEffect);
-        
+
         RefreshStatusEffects();
     }
-    
+
     public void RemoveStatusEffect(StatusEffect statusEffect) {
         activeStatusEffects.Remove(statusEffect);
-        
+
         RefreshStatusEffects();
     }
-    
-    public void ApplyTimedStatusEffect(StatusEffect statusEffect, Tower originTower)
-    {
+
+    public void ApplyTimedStatusEffect(StatusEffect statusEffect, Tower originTower) {
         TimedStatusEffect newTimedStatusEffect = new TimedStatusEffect(statusEffect, originTower);
-        
+
         bool statusEffectMissing = true;
-        
+
         foreach (TimedStatusEffect timedStatusEffect in timedStatusEffects) {
             if (newTimedStatusEffect.Equals(timedStatusEffect)) {
                 timedStatusEffect.RefreshTimer();
                 statusEffectMissing = false;
             }
         }
-        
+
         if (statusEffectMissing) {
             timedStatusEffects.Add(newTimedStatusEffect);
         }
-        
+
         RefreshStatusEffects();
     }
-    
+
     private void RefreshStatusEffects() {
```

Comment:
Could this be a performance issue later when we have many enemies with many effects and each time something is applied/removed it has to iterate through all effects again?